### DISCUSSION TITLE
GitHub Mobile

### DISCUSSION THREAD

**Discussion by Kaparrs** [community/community#36752] [Mobile]
It would be awesome if we could have more accessibility to options and other things insider the mobile app. Would boost productivity by a lot...

**Comment by ossanna16** [comment]
Thank you for the feedback @JajarGaming, I'm glad you found our community 🙂  Do you have any specific suggestions for options you would like to see in the mobile app?

**Reply by Kaparrs** [reply]
Ability to edit profile/settings. Edit files. Repository/Org management/settings
